communitiesuk/funding-service-design-assessment-store
# Fs 4315 cof export

### Change description
For COF R4W1 they want additional fields in the applicant info export (see ticket for the list). This PR:
- Adds those fields
- Adds unit tests for those fields
- Updates the `get_export_data` function to allow for new field types (postcode, sum of values)
- Updates the seed data script to work properly

---



### How to test
- Navigate to assessment
- Click 'Export Applicant Information' for COF R4W1
- View the CSV and check all fields hold correct values
